File: web/src/main/java/com/richo/reader/web/dto/DownloadJobStatus.java
package com.richo.reader.web.dto;

import java.time.LocalDateTime;

public class DownloadJobStatus
{
	public String lastRun;
	public final boolean running;

	public DownloadJobStatus(final LocalDateTime lastRun, boolean running)
	{
		this.lastRun = lastRun.toString();
		this.running = running;
	}
}

File: server/core/web/src/main/kotlin/com/richodemus/reader/web/dto/DownloadJobStatus.kt
package com.richodemus.reader.web.dto

import java.time.LocalDateTime

class DownloadJobStatus(lastRun: LocalDateTime, val running: Boolean, lastRunOutCome:String) {
    val lastRun: String = lastRun.toString()
    val lastRunOutCome:String = lastRunOutCome
}
